cmd/zc_processor.go

```diff
@@ -90,10 +91,34 @@ func (s *copyTransferProcessor) scheduleCopyTransfer(storedObject storedObject)
 				common.PanicIfErr(err)
```

Comment:
Why are we panicking here instead of just throwing appropriate error and terminating gracefully? We already have well-defined error here which we can return. I was going through the AzCopy code to find the usage of `json.Marshal()` and found that we use both - panic and throwing error.
```go
jsonOutput, err := json.Marshal(summary)
common.PanicIfErr(err)
```
and 

```go
b, err := json.Marshal(m)
if err != nil {
	return "", err
}
```
It makes sense to panic when unexpected wrong thing happens. In this case, we have well defined error which we can throw. Notice the return type of `scheduleCopyTransfer` method.

`func (s *copyTransferProcessor) scheduleCopyTransfer(storedObject storedObject) (err error)`

@zezha-msft your thoughts here ?

Reply:
@mohsha-msft I used common.panicIfError(err) because I am calling glcm.Dryrun() and returning the string that would be printed in cmd. This is similarly to how errors are handled in glcm.Progress()